### PR TITLE
Update session4 feb2023

### PR DESCRIPTION
**Pull request by luisesanmartin**

### CODE REVIEW

**Comment by luisesanmartin** [comment]
@weilu this is ready for review. No major changes, just a few adjustments based on the comments in #29. There was one suggestion I didn't implement because I don't with agree with it (the one about using `return` directly instead of `else`), but let me know if you have comments on that.
Thanks!

File: 1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb [hunk 1]
```diff
@@ -373,6 +373,17 @@
     "    return None, None"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run this line to verify that your solution works. It should not return an error\n",
```
Comment (weilu) [inline 1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb:382]: Consider add `%%script echo Remove this line after filling in your own code` at the top of this cell so the whole notebook can be executed without error.

File: 1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb [hunk 2]
```diff
@@ -1023,6 +1034,13 @@
     "    + `wb.data.DataFrame` returned a Pandas dataframe, which is very convenient for further data analysis\n",
     "- Many client libraries do much more than just retrieving the API results"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Final note:** Remember to go to `File` > `Save a copy in Drive` to save a copy of this notebook in your Google account."
```
Comment (weilu) [inline 1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb:1042]: Not sure we still want to say this now that we are running this session in databricks. For what is worth, to save the notebook it's `File > Export > IPython Notebook` in databricks

File: 1-foundations/4-api-and-dataviz/foundations-s4-dataviz.ipynb [hunk 3]
```diff
@@ -746,6 +746,13 @@
     "plt.ylabel('Number of countries')\n",
     "plt.savefig('histogram_seaborn.png');"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Final note:** Remember to go to `File` > `Save a copy in Drive` to save a copy of this notebook in your Google account."
```
Comment (weilu) [inline 1-foundations/4-api-and-dataviz/foundations-s4-dataviz.ipynb:754]: Ditto

File: 1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb (hunk 1 above)
Comment (luisesanmartin) [reply 1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb]: added in 09b9665

File: 1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb (hunk 2 above)
Comment (luisesanmartin) [reply 1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb]: added clarifying note in 09b9665

File: 1-foundations/4-api-and-dataviz/foundations-s4-dataviz.ipynb (hunk 3 above)
Comment (luisesanmartin) [reply 1-foundations/4-api-and-dataviz/foundations-s4-dataviz.ipynb]: added clarification in 09b9665

**Pull request merged**